admin_manual/ai/app_summarai.rst

```diff
@@ -0,0 +1,163 @@
+==========================================
+App: SummarAI (Talk chat summarize bot)
+==========================================
+
+.. _ai-app-summarai:
+
+The *SummarAI* app is one of the apps that utilize the llm or any other configured llm provider in Nextcloud and act as a configurable bot for `Nextcloud Talk`.
```

Comment:
`llm`,  at the first usage, it would be nice to have the full name (I suppose it is the large language model?), or already the local large language model (lllm, l3m??), because we should use it or any other llm provider....